web_development_101/project_pairing.md

```diff
@@ -29,6 +29,7 @@ Enough talk! Let's build!
 Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.  Don't forget to include your pair's github handle as well!
 
 * Add your solution below this line!
+ [SarfrazAnjum's solution](https://github.com/SarfrazAnjum/pomodoro-technique) - [View in Browser](https://github.com/SarfrazAnjum/pomodoro-technique/)
```

Comment:
Your `View in Browser` link does not redirect to a live version of the Pomodoro timer. Please remove or update the link.